segmentation/config.py

```diff
@@ -373,6 +372,10 @@ def check_modalities(ref_modalities, modalities, subject_name):
         val_subjects += subjects[end_train:end_val]
         test_subjects += subjects[end_val:]
 
+        self.logger.log(logging.INFO, f'{len(train_subjects)} subjects in the train set')
+        self.logger.log(logging.INFO, f'{len(val_subjects)} subjects in the validation set')
+        self.logger.log(logging.INFO, f'{len(test_subjects)} subjects in the test set')
+
         return train_subjects, val_subjects, test_subjects
```

Comment:
Add a line to specify shuffle and seeding
"Loading subject, whith shuffle=True seeding is None "
Can you add the path ( the path ) of the first subject with the modality name ?
for instance,
Train set : x subject the first one has /path1 for modality T1 /path2 for modality label ect ...